src/Call.tsx

```diff
@@ -873,40 +873,38 @@ export class Call extends EventEmitter {
   /**
    * CallMessage API is in beta.
    *
-   * Send {@link (CallMessage:class)}.
+   * Send a CallMessage.
    *
    * @example
    * To send a user-defined-message
    * ```typescript
-   * const message = new CallMessage({
-   *    content: { key1: 'This is a messsage from the parent call' },
-   *    contentType: 'application/json',
-   *    messageType: 'user-defined-message'
-   * })
-   * const outgoingCallMessage: OutgoingCallMessage = await call.sendMessage(message)
+   * const outgoingCallMessage: OutgoingCallMessage = await call.sendMessage({
+   *   content: { key1: 'This is a messsage from the parent call' },
+   *   contentType: 'application/json',
+   *   messageType: 'user-defined-message'
+   * });
    *
    * outgoingCallMessage.addListener(OutgoingCallMessage.Event.Failure, (error) => {
-   *    // outgoingCallMessage failed, handle error
+   *   // outgoingCallMessage failed, handle error
    * });
    *
    * outgoingCallMessage.addListener(OutgoingCallMessage.Event.Sent, () => {
-   *    // outgoingCallMessage sent
-   * })
+   *   // outgoingCallMessage sent
+   * });
    * ```
    *
-   * @param content - The message content
-   * @param contentType - The MIME type for the message.
-   * @param messageType - The message type.
+   * @param options The details of the CallMessage to send.
```

Comment:
Is this the reason why we converted CallMessage into a namespace? Can we change it back to an interface? Then update this line to `@param message - The details of the CallMessage to send.`